File: chess-board.py
def chess_board(input_num):
    board = []

    for row in range(input_num):
        current_row = ""

        for col in range(input_num):
            if (row + col) % 2 == 0:
                current_row += "W "
            else:
                current_row += "B "

        board.append(current_row)

    return "\n".join(board)


print(chess_board(5))
# print(chess_board(8))

File: chess-board.js
function chessBoard(inputNum) {
    let board = []

    for (let row = 0; row < inputNum; row += 1) {
        let currentRow = ""

        for (let column = 0; column < inputNum; column += 1) {
            if ((row + column) % 2 === 0) {
                currentRow += "W "
            } else {
                currentRow += "B "
            }
        }
        board.push(currentRow)
    }

    return board.join("\n")
}


console.log(chessBoard(5));
console.log(chessBoard(8));
